fuzzing/private/regression.bzl

```diff
@@ -48,7 +48,7 @@ exec '{engine_launcher}'
     runfiles = runfiles.merge(binary_info.engine_info.launcher_runfiles)
     return [DefaultInfo(executable = script, runfiles = runfiles)]
 
-fuzzing_regression_test = rule(
+fuzzing_regression_test_script = rule(
```

Comment:
I'm not sure I understand what is the reason for this change.

Reply:
`sh_test` supports coverage gathering on its C/C++ data dependencies (ref: https://youtu.be/P51Rgcbxhyk?t=599).

Reply:
That is true, but if we do this, one target becomes two targets, so we add one more target to all regression tests instantiations. I'm wondering if we can avoid that. Can we take any of the steps outlined at https://docs.bazel.build/versions/3.5.0/skylark/rules.html#code-coverage-instrumentation?